cloud-aws-native/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/metadata/AwsNativeMetadataCollector.java

```diff
@@ -155,11 +187,15 @@ public InstanceStoreMetadata collectInstanceStorageCount(AuthenticatedContext ac
                     .filter(ci -> instance.getInstanceId().equals(ci.getInstanceId()))
                     .findFirst()
                     .orElseThrow();
+            String instanceId = cloudInstance.getInstanceId();
```

Comment:
I have tried it, but the above(186-189) lines throw the 'no value present' error

Reply:
now works